exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,93 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "A basket containing only a single book is charged full price.",
+            "basket": [1],
+            "expected": 8.0
+         },
+         {
+	        "description": [
+			                "A basket containing only two of the same book is most inexpensively ",
+							"grouped as two, one book, groups."
+						   ],
+            "basket": [2, 2],
+            "expected": 16.0
+         },
+         {
+            "description": "No charge to carry around an empty basket.",
+            "basket": [],
+            "expected": 0.0
+         },
+         {
+	        "description": "A basket containing just two different books is the least expensive as a single group.",
+            "basket": [1, 2],
+            "expected": 15.2
+         },
+         {
+	        "description": "A basket of three different books is the least expensive as a single group.",
+            "basket": [1, 2, 3],
+            "expected": 21.60
+         },
+         {
+	        "description": "A basket of four different books is the least expensive as a single group.",
+            "basket": [1, 2, 3, 4],
+            "expected": 25.60
+         },
+         {
+	        "description": "A basket of five different books is the least expensive as a single group.",
+            "basket": [1, 2, 3, 4, 5],
+            "expected": 30.00
+         },
+         {
+	        "description": [
+		                    "For a basket containing eight books consisting of a pair each of the first three books ",
```

Comment:
This could be shortened to "Two groups of four is cheaper than a group of five plus a group of three". This tells us why we have this test - to catch a common mistake students might make (always make the largest group possible)

Can probably make similar changes for all others.